harness/determined/common/experimental/checkpoint/_checkpoint.py

```diff
@@ -24,6 +24,11 @@ class CheckpointState(enum.Enum):
 
 class Checkpoint(object):
     """
+    .. note::
+
+        The preferred path for checkpoint export APIs is
+        ``determined.experimental.client.get_checkpoint()``.
```

Comment:
I think the wording of these "preferred path" docs is unclear to me.

Can we make them say something like:

```
A Checkpoint object is usually obtained from ``determined.experimental.client.get_checkpoint()``.
```

I also don't think that it needs to be a `.. note::` block.